Tools/TestWebKitAPI/Tests/WebKit/WKBackForwardList.mm

```diff
@@ -311,6 +312,32 @@ - (void)webView:(WKWebView *)webView decidePolicyForNavigationAction:(WKNavigati
     EXPECT_STREQ([[list.currentItem URL] absoluteString].UTF8String, url3.absoluteString.UTF8String);
 }
 
+static bool navigated;
```

Comment:
It would be better to not introduce a new static global boolean.  Can we make this an ivar on the delegate and introduce a waitForSomething method on the delegate instead?

Reply:
Ok, done.